src/Raven.Client/Documents/Session/IAdvancedSessionOperations.Stream.cs

```diff
@@ -95,6 +106,50 @@ public partial interface IAdvancedSessionOperations
         ///     Returns the results of a query directly into stream 
         /// </summary>
         void StreamInto<T>(IRawDocumentQuery<T> query, Stream output);
+    }
+
+    public interface ITimeSeriesSessionStreamOperations<T>
+    {
+        IEnumerator<TimeSeriesStreamResult<T>> Stream(IQueryable<T> query);
```

Comment:
Wouldn't those methods cause ambiguity?

Reply:
No, since we pass the `T` here to be an explicit type.
```
 public partial interface IAdvancedSessionOperations : 
        IDocumentsSessionOperations,
        ITimeSeriesSessionStreamOperations<TimeSeriesAggregationResult>,
        ITimeSeriesSessionStreamOperations<TimeSeriesRawResult>,
        ITimeSeriesSessionStreamAggregationResultOperations,
        ITimeSeriesSessionStreamRawResultOperations
    {

    }
```